node_modules/oae-core/comments/css/comments.css

```diff
@@ -40,6 +40,15 @@
 
 /* iPhone resolution */
 @media (max-width: 480px) {
+    .comments-widget #comments-container > .media > .media-body {
+        overflow: visible;
+    }
+
+    .comments-widget .comments-level-0 .comments-reply-container,
```

Comment:
@sathomas : Yes, that was supposed to say "Note". I'm just suggesting that we include a fix for larger devices in this PR as well.